recurly/__init__.py

```diff
@@ -364,6 +372,11 @@ def create_shipping_address(self, shipping_address):
         url = urljoin(self._url, '/shipping_addresses')
         return shipping_address.post(url)
 
+    def child_accounts(self):
```

Comment:
I think this should also be handled by the relatiator?

Reply:
Confirmed, taking out this function does not change the behavior or fail the tests. I will remove it